_posts/2017-12-21-code-reviews.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Why do a code review?
+author: jake_hall
+external_url: http://blog.florxlabs.com/code-reviews
+---
+
+Code reviews are an integral part of the teams that I work on, and recently I was asked what value they added versus the time they took. Here are a few thoughts about why they are absolutely vital to any software development project.
+
+# What do I mean by a code review? #
+
+To us, a code review is the final check by the team that the change being delivered has the following attributes, before being merged into the main branch: 
+* there are no obvious issues with logic
+* follows agreed best practice, and coding standards
+* code is understandable, maintainable and clean
+* all automated tests are written, and have the necessary coverage
+* the acceptance criteria on the ticket has been met
```

Comment:
Should this be "have" instead of "has", since criteria is plural?

Reply:
Changed!